README.md

```diff
@@ -1,26 +1,31 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+# Overview
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+A Juju charm that deploys and manages haproxy on machine. HAProxy is a TCP/HTTP reverse proxy which is particularly suited for high availability environments. It features connection persistence through HTTP cookies, load balancing, header addition, modification, deletion both ways. It has request blocking capabilities and provides interface to display server status.
```

Comment:
Minor: reference the project https://github.com/haproxy/haproxy would be nice :)